packages/ui/src/theme/css/component/tabs.scss

```diff
@@ -0,0 +1,66 @@
+.amplify-tabs {
+  background-color: var(--amplify-components-tabs-background-color);
+  box-shadow: var(--amplify-components-tabs-box-shadow);
+  margin-bottom: var(--amplify-components-tabs-margin-bottom);
+}
+
+.amplify-tabs-item {
```

Comment:
I wanted to name this class `.amplify-tab-item`, but then we wouldn't be able to include it in the same `.js` file as `.amplify-tabs` for Style Dictionary.